.github/workflows/release-docker.yml

```diff
@@ -16,41 +16,48 @@ permissions:
   packages: write
 
 on:
+  workflow_call:
+    inputs:
+      package:
+       description: The package to release. _One_ of [dbt-core, dbt-redshift, dbt-bigquery, dbt-snowflake, dbt-spark, dbt-postgres]
+       required: true
+       type: string
   workflow_dispatch:
     inputs:
       package:
        description: The package to release. _One_ of [dbt-core, dbt-redshift, dbt-bigquery, dbt-snowflake, dbt-spark, dbt-postgres]
        required: true
+       type: string
```

Comment:
You can make this [`type: choice`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#:~:text=%27warning%27%0A%20%20%20%20%20%20%20%20type%3A-,choice,-options%3A%0A%20%20%20%20%20%20%20%20%2D) to make manual submission easier.